backend/docs/buildingsV1.yml

```diff
@@ -112,22 +124,21 @@ components:
           type: number
           format: float64
           description: longitude
-
-    Room:
-      type: object
-      properties:
-        name:
-          type: string
-        level:
-          type: integer
+          
     Floor:
       type: object
       properties:
         name:
           type: string
         level:
           type: integer
-        Room:
+        rooms:
```

Comment:
I think what I meant wrt the previous review was your reasoning as to why we should specify a Room object by itself and refer to the Room object under `rooms` or, as shown in the file right now, define the room object within the Floor object? Should we consider if we want to use the Room object in a different context other than coupled with a Floor?